src/main/java/com/massivecraft/factions/cmd/CmdPowerBoost.java

```diff
@@ -74,6 +97,19 @@ public void perform(CommandContext context) {
         }
     }
 
+    protected class PowerBoostBrigadier implements BrigadierProvider {
+        @Override
+        public ArgumentBuilder<Object, ?> get(ArgumentBuilder<Object, ?> parent) {
+            ArgumentCommandNode<Object, ?> generic = RequiredArgumentBuilder.argument("p/f/player/faction", StringArgumentType.word())
```

Comment:
Here we could use Brigadier's literal to show p/f/player/faction in different lines, but because it's not implemented anywhere else I didn't include it.